plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1077,6 +1120,13 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
             } );
 
       my->accepted_block_connection = my->chain->accepted_block.connect( [this]( const block_state_ptr& blk ) {
+         if (auto dm_logger = my->chain->get_deep_mind_logger()) {
+            fc_dlog(*dm_logger, "ACCEPTED_BLOCK ${num} ${blk}",
+               ("num", blk->block_num)
+               ("blk", blk)
```

Comment:
We log the full block state to avoid having to reconstruct it so we reduce the chance of differences, specially with the deferred handling that needs to be made correctly. It's much easier to log the full block state than taking the chance. And we also only log the transaction_trace for now in applied_transaction, so there is info missing like the transaction signatures, which are present in the `block_state`.

It's something that we could change in the future to reduce the amount of data transferred. I'll take not of this for a future improvement, but for now, I would prefer to avoid changing it if you agree.

Reply:
It does seem like something you would want to address in the future. It is fine for now.